src/services/ipfs.ts

```diff
@@ -1,77 +1,89 @@
-// import { Client } from "../client";
-// import { FormData } from 'form-data';
+import { type Client } from '../client';
 
-// export enum IpfsContentFormat {
-//   FormData = 'formdata',
-//   ArrayBuffer = 'arraybuffer',
-//   Blob = 'blob',
-//   Text = 'text',
-//   JSON = 'json'
-// }
+export enum IpfsContentFormat {
+    FormData = 'formdata',
+    ArrayBuffer = 'arraybuffer',
+    Blob = 'blob',
+    Text = 'text',
+    JSON = 'json',
+}
 
-// export class IpfsService {
-//     constructor(private client: Client) {}
+export class IpfsService {
+    constructor (private readonly client: Client) {}
 
-//     upload = async (obj: object): Promise<string> => {
-//       try {
-//         const stringified = JSON.stringify(obj)
-//         const blob = new Blob([stringified], { type: 'application/json' })
-//         const formData = new FormData()
-//         const blobText = await blob.text()
-//         formData.append('file', blob)
+    upload = async (obj: object): Promise<any> => {
+        try {
+            let blob = new Blob([JSON.stringify(obj)], { type: 'application/json' });
```

Comment:
@djmbritt Things worked well for me if I just switched to the global `Blob` object instead of the complicated import statement that was happening. Both in the tests and in react-native this works, am I missing something?

Reply:
You're right, I read online that the only way to access the Blob constructor was through the `node:Buffer` import.
I missed this one, it's much cleaner.